generators/app/templates/default/src/build.cake

```diff
@@ -17,6 +17,7 @@ Sitecore.Constants.SetNames();
 Sitecore.Parameters.InitParams(
     context: Context,
     msBuildToolVersion: MSBuildToolVersion.Default,
+    supportHelix20: "<%= supportHelix20X %>",
```

Comment:
Please unwrap from quotes

Reply:
This parameter will be converted from string to boolean later